composer/spec/dependabot/composer/file_updater/lockfile_updater_spec.rb

```diff
@@ -691,31 +691,6 @@
       end
     end
 
-    context "when there are patches (composer v2)" do
```

Comment:
patch data isn't going to be re-applied to composer.lock

https://github.com/dependabot/dependabot-core/pull/6620